data/world/critters/duck/init.lua

```diff
@@ -22,4 +22,6 @@ world:new_critter_type{
       remove = { "remove" },
    },
    animations = animations,
+   size = 1,
+   reproduction_rate = 80,
```

Comment:
Ducks are hard to hunt, because they're in water. We have no water carnivores. So, their reproduction rate needs to be very low.

Reply:
Done.
P.S. When a duck is on a shore field, a carnivore going there will consider eating it